src/main/java/com/synopsys/integration/alert/channel/email/descriptor/EmailDistributionDescriptorActionApi.java

```diff
@@ -100,7 +101,7 @@ public TestConfigModel createTestConfigModel(final FieldModel fieldModel, final
             blackDuckProjectEntities
                 .stream()
                 .forEach(project -> {
-                    final Set<String> emailsForProject = emailGroupChannel.getEmailAddressesForProject(project, projectOwnerOnly);
+                    final Set<String> emailsForProject = emailAddressHandler.getBlackDuckEmailAddressesForProject(project, projectOwnerOnly);
```

Comment:
This call will need to be specific to provider in the future, correct? If we require a provider to define its own email address functionality, then we can call the provider method here as well.